### ISSUE TITLE
brainfuck example is not a fork bomb

### ISSUE THREAD

**Issue by Mee42** [aaronryank/fork-bomb#18]
it is an infinite loop. If you were to convert it to C, it would look like this:
```c
void run(int* mem) {
    *mem++;
    while(*mem) { mem++; *mem++; };
}
```
which is very clearly not a fork bomb

**Comment by Aritz331** [comment]
# What?
# idk why it is but as u can c no one answers lol
### its strange but idk why you say that but idk 2 much languages, only a bit of vbs and a bit more of batch

**Comment by Aritz331** [comment]
# If there is no activity on this issue in months, a bot will tag it and time later the bot will close it, because it happened to me

**Comment by Mee42** [comment]
Please don't yell

No one answered because the maintainer isn't maintaining this project, and no one other then the maintainer would give af

**Comment by ghost** [comment]
This is correct, it is not a correct brainfuck fork bomb, I have come across a solution (not) by myself, it is not a fork bomb but it leaves traces.
```
+[[+]>+]
```

**Comment by ghost** [comment]
This might cause a system to crash if left long. `+[[.+]+]

**Comment by Xprogrammer777** [comment]
> This is correct, it is not a correct brainfuck fork bomb, I have come across a solution (not) by myself, it is not a fork bomb but it leaves traces.
> 
> ```
> +[[+]>+]
> ```

btw brainfuck is only brainfuck
